cmd/root/io.go

```diff
@@ -21,7 +21,8 @@ func initOutputFlag(cmd *cobra.Command) *outputFlag {
 	// Configure defaults from environment, if applicable.
 	// If the provided value is invalid it is ignored.
 	if v, ok := env.Lookup(cmd.Context(), envOutputFormat); ok {
-		f.output.Set(v)
+		// QQQ log the error?
```

Comment:
We don't have access to a logger yet. If the value is bad it will be ignored.

Reply:
I have removed the comment and added //nolint:errcheck annotation.